routingv8/routes_test.go

```diff
@@ -0,0 +1,89 @@
+package routingv8_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"go.einride.tech/here/routingv8"
+	"gotest.tools/v3/assert"
+)
+
+type RoutesMock struct {
+	responseStatus int
+	responseBody   routingv8.RoutesResponse
+}
+
+func (c *RoutesMock) Do(req *http.Request) (*http.Response, error) {
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json")
+	b, err := json.Marshal(c.responseBody)
+	if err != nil {
+		return nil, err
+	}
+	r := bytes.NewReader(b)
+	return &http.Response{
+		StatusCode:    c.responseStatus,
+		Header:        headers,
+		Body:          io.NopCloser(r),
+		ContentLength: int64(len(b)),
+	}, nil
+}
+
+func TestRoutingervice_Routes(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	// Einride Gothenburg.
+	origin := routingv8.GeoWaypoint{
+		Lat:  57.707752,
+		Long: 11.949767,
+	}
+	// Einride Stockholm.
+	destination := routingv8.GeoWaypoint{
+		Lat:  59.337492,
+		Long: 18.063672,
+	}
+
+	exp := routingv8.RoutesResponse{
+		Routes: []routingv8.Route{
+			{
+				ID: "route-1",
+				Sections: []routingv8.Section{
+					{
+						ID:   "section-1",
+						Type: "veicle",
+						Departure: routingv8.Place{
+							Type:             "place",
+							Location:         origin,
+							OriginalLocation: origin,
+						},
+						Arrival: routingv8.Place{
+							Type:             "place",
+							Location:         destination,
+							OriginalLocation: origin,
```

Comment:
Minor thing because I guess it won't have any effect on the test, but I'm guessing it would make more sense to have destination here as well? I'm guessing it's a field describing what the originally indicated origin was, while the Location field above it describes what HERE slightly adjusted the place to be - perhaps because the OriginalLocation wasn't exactly on a road? Do you happen to know.
```suggestion
							OriginalLocation: destination,
```

Reply:
I made the assumption that originalLocation would be the origin but I looked at the documentation again and it seems like you are correct